saleor/core/utils/random_data.py

```diff
@@ -203,9 +205,19 @@ def create_product_variants(variants_data):
             continue
         defaults['product_id'] = product_id
         defaults['attributes'] = json.loads(defaults['attributes'])
+        defaults['price_override'] = get_in_default_currency(
+            defaults, 'price_override', settings.DEFAULT_CURRENCY)
+        defaults['cost_price'] = get_in_default_currency(
+            defaults, 'cost_price', settings.DEFAULT_CURRENCY)
         ProductVariant.objects.update_or_create(pk=pk, defaults=defaults)
 
 
+def get_in_default_currency(defaults, field, currency):
+    if dict.get(defaults, field):
+        return Money(getattr(dict.get(defaults, field), 'amount'), currency)
```

Comment:
At this point, we know that `field` is present in `defaults`. Also, there's no need to do `getattr` as we will always have a `Money` type here. A Pythonic way to write this would be:
```
return Money(defaults[field].amount), currency)
```

Reply:
@maarcingebala @NyanKiyoshi 

I have a syntactical error:

```text
E       return Money(defaults[field].amount), currency)
E                                                     ^
E   SyntaxError: invalid syntax
```

Can you please advise me how to correct it - thanks !

```python
def get_in_default_currency(defaults, field, currency):
    if field in defaults:
        return Money(defaults[field].amount), currency)
    return None
```

Reply:
Comma after `amount` should is unnecessary, it should be:
`return Money(defaults[field].amount, currency)`

Reply:
oops, i missed that - you mean the parenthesis. thanks.

Reply:
Yep, sorry - I was thinking of that closing bracket 😄.